packages/code.rb

```diff
@@ -35,13 +33,15 @@ class Code < Package
   ENV['PATH'] = "#{ENV['HOME']}/.nodebrew/current/bin:#{ENV['PATH']}"
 
   def self.build
-    node_old = `nodebrew ls | fgrep 'current: ' | cut -d' ' -f2`
+    node_old = `nodebrew ls | fgrep 'current: ' | cut -d' ' -f2 | grep -v none`.chomp
     system "nodebrew install 8.15.0 || true"
     system "nodebrew use 8.15.0"
     system "npm install gulp"
     system "yarn install"
     system "yarn run gulp vscode-linux-#{@arch}"
-    system "nodebrew use #{node_old}"
+    if node_old != ''
```

Comment:
You don't need the ` | grep -v none`.  You could:
`system "nodebrew use #{node_old}" unless "#{node_old}" == "none"`
Also, another consideration is to remove this node version when complete, therefore not impeding on the users node space:
`system "nodebrew uninstall 8.15.0" unless "#{node_old}" == "v8.15.0"`

Reply:
You might want to use a variable for "v8.15.0" also.

Reply:
Good idea. That sounds perfect.

Reply:
One step ahead of you there, @uberhacker.

Reply:
How about?
```
    system "nodebrew install #{node_ver}" unless "#{node_ver_installed}" == "#{node_ver}"
    system "nodebrew use #{node_ver}" unless "#{node_old}" == "#{node_ver}"
```

Reply:
That works too.